api/Makefile

```diff
@@ -135,8 +135,11 @@ protoc/system/v1alpha1:
 build: ## Build generated files
 	go build ./...
 
+GO_TEST ?= go test
+GO_TEST_OPTS ?=
+
 test: ## Run tests
-	go test ./...
+	$(GO_TEST) $(GO_TEST_OPTS) -race -covermode=atomic -coverpkg=./... -coverprofile="$(COVERAGE_PROFILE)" ./...
```

Comment:
This is good for consistency. Then again if this make target exists, can we just invoke it instead of `go test ./api/...`?

Reply:
I'm sorry I don't think I get your point there

Reply:
Never mind, let's just leave it as it is and stick with the proposal for parametrised Makefile target.